docs/DevGuide.rst

```diff
@@ -68,10 +60,10 @@ Now you can update your fork on a regular basis with changes from the original r
    git fetch upstream
 
    # Merge changes into your local branch
-   git checkout --track origin/Dev
-   git merge upstream/Dev
+   git checkout --track origin/master
+   git merge upstream/master
 
-This will for example allow you to keep the development branch ``Dev`` up to date. Doing this will prevent an unpleasent surprise once you're ready to submit your pull request since there will be less or no merge conflicts.
+This will for example allow you to keep the development branch ``master`` up to date. Doing this will prevent an unpleasent surprise once you're ready to submit your pull request since there will be less or no merge conflicts.
```

Comment:
Probably it is better to simply write "keep the master branch up to date", as we do not have a dedicated development branch anymore. This applies also in later parts of this file.

Reply:
I incorporated your suggestion, is it ok now?